packages/@expo/cli/src/export/__tests__/resolveOptions-test.ts

```diff
@@ -46,7 +40,7 @@ describe(resolveOptionsAsync, () => {
     await expect(
       resolveOptionsAsync('/', { '--platform': ['android', 'all'] })
     ).resolves.toMatchObject({
-      platforms: ['android', 'ios'],
+      platforms: ['android', 'ios', 'web'],
```

Comment:
As `web` uses Metro, do we expect `all` to expand to `web`?

Reply:
no, because `web` is an out of tree platform and requires extra installation.

Reply:
oh I see, this is for export––yes it should include all metro platforms.